### ISSUE TITLE
InfluxDB / Homebrew issue with M1 processor

### ISSUE THREAD

**Issue by msonderegger** [MontrealCorpusTools/PolyglotDB#171]
System Info
<img width="223" alt="Screen Shot 2022-06-30 at 4 27 49 PM" src="https://user-images.githubusercontent.com/798417/176772491-966a6843-5b4f-4b34-b81e-a86531436a33.png">
:

homebrew now installs by default into `/opt/homebrew` on Apple Silicon machines (like this one, M1 Pro processor).  From Homebrew docs:

![Uploading Screen Shot 2022-06-30 at 4.29.50 PM.png…]()

This required me changing the `pgdb` script referenced [here](https://polyglotdb.readthedocs.io/en/latest/getting_started.html#set-up-local-database) .  Diff of changes:

<img width="771" alt="Screen Shot 2022-06-30 at 4 31 53 PM" src="https://user-images.githubusercontent.com/798417/176773104-4f082e86-869e-403e-806a-cb5b1ece0f12.png">

... after which the polyglot install worked.  (I forget what the `arch -arm` call is for, I found it online somewhere..)

So I think this script just has to be changed to check if the machine uses Apple Intel or Apple Silicon chip, use the current code (    elif sys.platform == 'darwin'...)
for Apple Intel, add a new block for Apple Silicon?

**Comment by msonderegger** [comment]
@michaelhaaf if there is any way we can prioritize this, it's driving me nuts :) I now can't install polyglot on my laptop using (`conda`), it has to be from source, which leads to other issues.

**Comment by michaelhaaf** [comment]
Yep, this can be prioritized. I'll message on slack for a few more details

**Comment by msonderegger** [comment]
Thank you !
